### ISSUE TITLE
Stage issue: add1 alpha parameters

### ISSUE THREAD

**Issue by K4thos** [Windblade-GR01/Ikemen-GO#202]
Stage: [Taiyo.zip](https://github.com/Windblade-GR01/Ikemen-GO/files/5619255/Taiyo.zip)

Mugen 1.1:
![mugen008](https://user-images.githubusercontent.com/8928195/100676400-1ad9dc00-3369-11eb-8d03-5788af2c9dc2.png)
Ikemen GO:
![ikemen011](https://user-images.githubusercontent.com/8928195/100676418-25947100-3369-11eb-8791-3955c7d3d7af.png)

Element responsible for this problem:
```ini
[BG CloudsMassive]
type=normal
spriteno=0,2
scalestart=.75,.35
start=0,-50
delta=.08,.4
tile=1,0
velocity=.035,0
trans=add1
alpha=64,256
```

As presented in above screenshot this stage also displays players shadows differently:

```ini
[Shadow]
intensity = 64
color = 128,128,128
yscale = .5
fade.range = 0,0
```

**Comment by K4thos** [comment]
Shadows problem in this stage was due to changes introduced in mugen 1.1, where color parameter is completely ignored (only intensity is used). In mugen 1.0 both were used at the same time, so I've implemented 1.1 behaviour checking if stage mugenversion = 1.1.

**Comment by K4thos** [comment]
Based on my tests add1 doesn't need special alpha handling, when alpha parameter is specified. Verified on stage backgrounds (this ticket) and Skadi by 雨树樱枫险FXNULL char available here: https://twitter.com/mugen_fxnull

https://user-images.githubusercontent.com/8928195/148865425-c5557682-8c8a-40af-9ce9-b8bbd5974f34.mp4

Should be fixed in above commit, although needs testing if this change doesn't lead to regression.
